_posts/2020-03-26-route-ocp-outage.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title: "OpenShift Routing Issue"
+date: 2020-03-26 12:34:00 +0000
+categories: openshift outage update
+---
+
+**What Happened**
+
+At ~11:30am today we became aware of an issue causing system wide service degradation to the OpenShift Container Platform (OCP) as well as hosted applications and shared services such as SSO and RocketChat. Through investigation it was determined the issue was caused by the same router pod (HAProxy) issue that we had several days ago. 
```

Comment:
11:00 am a user reported slowness. I started investigating and found it was the router issue. I then tried to move the router traffic to another node, and found it was also crashing at approx 11:10. Now everything was down.